pydid/doc/builder.py

```diff
@@ -102,19 +83,6 @@ def _default_id_generator(self):
         )
         yield from _default_id_generator(self._id_base, start=start)
```

Comment:
should this be `self._id_generator(`? Using two different id generators will create unintentionally duplicate ids.

Reply:
This is saying "yield from the global _default_id_generator initialized with these values" so delegating to a generator instance that is created in that method

Reply:
i.e. `_default_id_generator` from line 13 vs `self._default_id_generator`